lib/cli/run.js

```diff
@@ -258,6 +258,16 @@ exports.builder = yargs =>
         }
       });
 
+      types.boolean.concat(types.string, types.number).forEach(opt => {
+        if (Array.isArray(argv[opt])) {
+          throw createInvalidArgumentValueError(
+            `Argument "${opt}" can only be provided once`,
+            opt,
+            argv[opt]
+          );
+        }
+      });
+
```

Comment:
Further, convert `slow` and `timeout` back to numeric using `ms`.

Reply:
This happens within `Suite#timeout`, so I didn't do it here.